python/lsst/meas/mosaic/utils.py

```diff
@@ -57,7 +57,7 @@ def checkHscStack(metadata):
     coordinate system conventions that differ between the two.
     """
     try:
-        hscPipe = metadata.get("HSCPIPE_VERSION")
+        hscPipe = metadata.getArray("HSCPIPE_VERSION")
```

Comment:
Not sure if this is useful, but I poked around on tiger for old hsc calexp, and found ```md.get("HSCPIPE_VERSION")
'4.0.2' 
md.get("HSCPIPE_VERSION", True)
['4.0.2']```

Reply:
I think I messed this one up -- it should almost certainly be getScalar